marconi-mamba/db-utils/exe/Main.hs

```diff
@@ -29,13 +18,11 @@ cliParser = CliOptions
                               <> metavar "FILENAME"
                               <>  showDefault
                               <> value "./.marconidb/4/utxo-db"
-                              <> help "Path to the utxo database.")
-    <*> pNetworkId
```

Comment:
we were using networkid for doing address conversion and it no longer is needed in the db utils. 
the db-utils is used to get the most frequently used addresses so that we can run tests/demo